Includes/langMenu.cpp

```diff
@@ -1,11 +1,13 @@
 #include "langMenu.hpp"
+#include <algorithm>
 
 #ifdef NXDK
 #include <xboxkrnl/xboxkrnl.h>
 #endif
 
 static
 std::vector<std::pair<std::string, uint32_t>> languages{
+  {"Unknown", 0},
```

Comment:
Whats the reason for not using an classical enum?

Reply:
Mainly that the names are displayed in the language selection menu and printing variable names is not easily done.

Reply:
I see! So why not combine our ideas: 
`std::vector<std::pair<std::string, enum language>>` (I have no idea about c++ just so you get my idea)
No magic numbers in the code & printable strings!